src/Network/Network/AzureFirewallPolicy/NatRule/NewAzureFirewallPolicyNatRuleCommand.cs

```diff
@@ -79,6 +79,17 @@ public override void Execute()
         {
             base.Execute();
 
+            // One of SourceAddress or SourceIpGroup must be present
+            if ((SourceAddress == null) && (SourceIpGroup == null))
+            {
+                throw new ArgumentException("Either SourceAddress or SourceIpGroup is required.");
+            }
+
+            if (DestinationAddress == null)
+            {
```

Comment:
if it's required, please make the parameter mandatory

Reply:
Makes sense. Actually it is already mandatory so removed this check.